frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -119,10 +102,6 @@ const FileHandler = ({
     const parsedName = activeMembership?.parsedName;
     const senderName = activeMembership?.service;
     const client = `${parsedName}.${senderName}`;
-    const validateOnly = useMemo(
```

Comment:
Just wanted to point out that I got rid of a few `useMemo`s in this changeset -- we don't need to memoize inferred values if they're not complex to recompute on re-renders.  And most inferred values aren't complex to recompute 🧹